@xen-orchestra/lite/src/components/tasks/TasksTable.vue

```diff
@@ -7,26 +7,45 @@
       <th>{{ $t("task.started") }}</th>
       <th>{{ $t("task.estimated-end") }}</th>
     </template>
-
+    <tr v-if="!hasTasks">
+      <td colspan="5" class="no-tasks">{{ $t("no-tasks") }}</td>
+    </tr>
```

Comment:
I think it would be better if it was handled by default in `TaskRow` or maybe in `UiTable`

Reply:
`TaskRow` is a component which displays a single row for a given task, it can't know whether the parent has data or not and it wouldn't make sense to call TaskRow with no data.

And `UiTable` is to generic to handle that. It is not aware of children data.

For reusability, I propose having a `NoDataRow` component.

It would be used like this:

```vue
<NoDataRow v-if="!hasData" colspan="5">No data</NoDataRow>
```

Which would produce this (with the according CSS):
```html
<tr class="no-data-row">
  <td class="colspan="5">No data</td>
</tr>
```